packages/interface-default/src/index.tsx

```diff
@@ -122,47 +155,59 @@ export const UnobtrusiveConsentControlUI: React.FC<UnobtrusiveConsentControlUIPr
           slideDuration={slideDuration}
         />
       )}
-      <Form
-        onSubmit={onSubmitCb}
-        initialValues={initial}
-        render={({ handleSubmit }) => (
-          <>
-            {showForm && (
-              <Anime
-                duration={slideDuration}
-                translateY={['10%', '-100%']}
-                easing="easeInOutQuad"
-              >
-                <form onSubmit={handleSubmit} className={clsx(styles.pane)}>
-                  <div className={clsx(styles.content)}>
-                    <h2>Website Settings</h2>
-                    <p>
-                      Some features are disabled by default to protect your
-                      privacy:
-                    </p>
-                    {integrations.map(
-                      (integration: IntegrationConfigOptions) => (
-                        <Integration
-                          key={integration.id}
-                          Switch={Switch}
-                          {...integration}
-                        />
-                      )
-                    )}
-                    <SubmitButton>Save</SubmitButton>
-                  </div>
+      {renderBackdrop && (
+        <Backdrop
+          show={showForm}
+          fadeDuration={slideDuration}
+          styles={styles}
+        />
+      )}
+      <CSSTransition
```

Comment:
I guess its time to make the whole form a component and make it overridable like the others